### PR TITLE
Option to set order of processors

### PR DESCRIPTION
**Pull request by Erikvv**
Currently you cannot determine the order in which processors are added, and thus called.

In other parts of symfony this is done through a "priority" attribute.

I have added this and I'm using this in a proprietary project.

If it is accepted I will also make a PR for the documentation

### CODE REVIEW

File: DependencyInjection/Compiler/AddProcessorsPass.php [hunk 1]
```diff
@@ -28,32 +28,57 @@ public function process(ContainerBuilder $container)
...
+            $processors,
+            function (array $left, array $right) {
+               return $left['tag']['priority'] - $right['tag']['priority'];
+            }
+        );
+
+        foreach ($processors as $processor) {
+            $tag = $processor['tag'];
+            $id = $processor['id'];
+
```
Comment (Erikvv) [inline DependencyInjection/Compiler/AddProcessorsPass.php:58]: Below this line the code is identical to what it was, except 1 level less indented

**Comment by natepage** [comment]
Any news on this PR, we have a project that requires this feature. Anything we can do to help make it move forward?

**Review by jderusse — CHANGES_REQUESTED** [review]
Symfony exposes a `PriorityTaggedServiceTrait` trait that provides all the logic you need in this PR.

This would also deserve an entry in the CHANGELOG

**Comment by Erikvv** [comment]
I am no longer in the ecosystem. Someone else can make a new PR.

**Pull request closed without merging**